libraries/botbuilder-core/botbuilder/core/bot_adapter.py

```diff
@@ -14,7 +14,7 @@
 
 class BotAdapter(ABC):
     BOT_IDENTITY_KEY = "BotIdentity"
-    BOT_OAUTH_SCOPE_KEY = "OAuthScope"
+    BOT_OAUTH_SCOPE_KEY = "botbuilder.core.BotAdapter.OAuthScope"

```

Comment:
Not sure what the standard is for python but we seem to be mixing capitalization conventions here, is either all camelCase (microsoft.bot.builder.botAdapter.oAuthScope) or all PascalCase (Microsoft.Bot.Builder.BotAdapter.OAuthScope) but this doesn't seem right.

Reply:
botbuilder.core is the package name.  They are all lower case.  Regardless... this is just a string key.   It could be anything and doesn't have to follow any convention.